swan-lake/development-tutorials/ballerina-persist/persist-cli-tool.md

```diff
@@ -125,13 +134,14 @@ This command includes the following steps,
 You can use the `bal persist generate` command to generate the derived types, client, and script files.
 
 ```bash
```

Comment:
```suggestion
```
```

Remove `bash` to avoid incorrect syntax highlighting in the code

Reply:
Fixed in [5e81976](https://github.com/ballerina-platform/ballerina-dev-website/pull/9246/commits/5e81976916a09b9206b6f718416408d5686ab873)